Airports/EGBB/Ownership.txt

```diff
@@ -15,45 +15,45 @@ OWNER:BBR
 
 SECTOR:BBTWR:0:2000
 OWNER:BBT:BBR:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:BBT:BBR:LC:LNC:LWC:O:LM:LSC:L
-ALTOWNER:Observing LON_C_CTR:BBT:BBR:O:LM:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing London FIR:O:LM:BBT:BBR:LC:LNC:LWC:LSC:L
-ALTOWNER:Mentoring TWR/APP:BBTM:BBT:BBRM:BBR:LC:LNC:LWC:LSC:L
+ALTOWNER:Observing LON_SC_CTR:BBT:BBR:COWLY:TCM:LCW:LC:LNC:LWC:O:LM:LSC:L
+ALTOWNER:Observing LON_C_CTR:BBT:BBR:O:LM:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
```

Comment:
The inserted bit needs to come _before_ LM so that if TC Midlands is online whilst someone is observing LON_C_CTR the airspace that TCM owns doesn't show to them. 
```suggestion
ALTOWNER:Observing LON_C_CTR:BBT:BBR:O:COWLY:TCM:LCW:LM:LC:LNC:LWC:LSC:L
```